File: 2020年11月/最大连续1的个数.c
int findMaxConsecutiveOnes(int* nums, int numsSize) {
    int i;
    int count=0;
    int t=0;
    for(i=0;i<numsSize;i++){
        if(nums[i]==1){
            count++;
        }
        else{
            if(count>t)
                t=count;
            count=0;
        }
    }
    return count>t?count:t;
}

File: 2020年11月/最大连续1的个数.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        maxnum=0
        count=0
        for i in range(len(nums)):
            if nums[i]==1:
                count+=1
            else:
                if count>maxnum:
                    maxnum=count
                count=0
        if count>maxnum:
            maxnum=count
        return maxnum